Integrations/Slack/Slack_test.py

```diff
@@ -1419,6 +1521,208 @@ def fake_clean(text, client):
     assert entry_args['footer'] == '\n**From Slack**'
 
 
+def test_check_for_answers(mocker, requests_mock):
+    import Slack
```

Comment:
why are we importing slack in every function ?

Reply:
In order to mock functions from within the integration